osx/cookbooks/vim/default.rb

```diff
@@ -0,0 +1,7 @@
+package 'vim' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.